### PR TITLE
Add Suno icon

### LINKED ISSUE #11149: Request: Suno
### Brand Name

Suno 

### Website

https://suno.com/

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by anhthang**
image

**Issue:** closes #11149 

**Popularity metric:**

Global Rank: 675

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

SVG extracted from the website

### CODE REVIEW

**Comment by uncenter** [comment]
> The Suno name and logos are trademarks and service marks of Suno (collectively the "Suno Trademarks").... Nothing in these Terms of Service or the Service should be construed as granting... any license or right to use any of Suno Trademarks displayed on the Service, without our prior written permission in each instance.

Looks like we will need to request permission. @anhthang would you be willing to reach out? Apologies for not checking this in the earlier issue!

**Comment by anhthang** [comment]
np, i just sent them email to request for permission

**Comment by anhthang** [comment]
@uncenter they replied, i think we can ship it now

image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15182,6 +15182,11 @@
             "source": "https://sites.google.com/sumologic.com/sumo-logic-brand/home",
             "guidelines": "https://sites.google.com/sumologic.com/sumo-logic-brand/home"
         },
+        {
+            "title": "Suno",
+            "hex": "FFFFFF",
```
Comment (uncenter) [inline _data/simple-icons.json:15187]: My only question is I can't figure out / decide if we should be using `#000000` here or `#ffffff`... 🤔

Comment (anhthang) [reply _data/simple-icons.json:15187]: I'm also finding it challenging to decide. On their website, in some areas, they use a white logo within a black rounded square, while the header uses a black logo. But changing to `#000000` is not a problem I think.

Comment (uncenter) [reply _data/simple-icons.json:15187]: I think let's change to `#000000`:
suno

Comment (anhthang) [reply _data/simple-icons.json]: @uncenter updated

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
Great work @anhthang, thank you! ❤️

**Pull request merged**